text/0061-error-format-and-definitions.md

```diff
@@ -468,6 +468,19 @@ HTTP Code: `400 Bad Request`
 - The `:attribute` is inferred when the message is generated.
 - The `:filterableAttributes` is inferred when the message is generated. It contains the list of filterable attributes separated by a comma. `filterableAttribute1, filterableAttribute2, ...`
 
+#### Variant: Filtering on a non filterable attribute when `filterableAttributes` is empty
+
+```json
+{
+    "message": "Attribute `:attribute` is not filterable. This index does not have configured filterable attributes.",
+    "code": "invalid_filter",
+    "type": "invalid_request",
+    "link": "https://docs.meilisearch.com/errors#invalid_filter"
```

Comment:
I'm not sure why, but this works: `https://docs.meilisearch.com/reference/api/error_codes.html#invalid-filter` not `https://docs.meilisearch.com/reference/api/error_codes.html#invalid_filter`.

Any thoughts @guimachiavelli @bidoubiwa ?

Reply:
Good spot @brunoocasali! May need to update the whole specification then 🥷 At least the link leads to the errors page.

Reply:
The errors page story is an old one built on an ancient and questionable script. See https://github.com/meilisearch/documentation/issues/1299
But good new, it is being updated by @maryamsulemani97 https://github.com/meilisearch/documentation/pull/1503

Once that PR is merged there shouldn't be any anchor problems and we'll be able to search for error codes in the search bar of the documentation

Reply:
As long as the anchors work and the routes returned by the binary do not fall on a 404, it is all good on our side. 

Please, let us know if we need to make a change on our end. On the other side, it could be managed with redirections on the documentation site 😇

Reply:
The Errors page was until recently completely separate from the regular docs website. While all our docs are written in markdown, that specific page was generated from a `yaml` file as described by @bidoubiwa. We scrapped that a few weeks ago and moved the Errors page completely inside vuepress. The transition was a glorified copypaste of `yaml` content into an `md` file; it also moved `docs.meilisearch.com/errors` into `docs.meilisearch.com/api/error_codes.html`. https://github.com/meilisearch/documentation/pull/1503 will further improve the page (and possibly move it again). 

The anchor link formatting is related to this transition. Vuepress automatically turns spaces into `-`, not `_`. `docs.meilisearch.com/errors#invalid_filter` only worked because the Errors page was outside vuepress. Now that the Errors page is being parsed as markdown, all anchor links will use `-`.